### ISSUE TITLE
{[Error: EISDIR: illegal operation on a directory, read] errno: -21, code: 'EISDIR', syscall: 'read' }

### ISSUE THREAD

**Issue by zakirsajib** [Dexus/pem#76]
```
npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "autoprefixer"
npm ERR! node v5.5.0
npm ERR! npm  v3.8.0
npm ERR! code ELIFECYCLE
npm ERR! pem@1.0.0 autoprefixer: `postcss -u autoprefixer -r static/dist/css/*`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the pem@1.0.0 autoprefixer script 'postcss -u autoprefixer -r static/dist/css/*'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the pem package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     postcss -u autoprefixer -r static/dist/css/*
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs pem
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls pem
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Library/WebServer/Documents/pem/wp-content/themes/pem/npm-debug.log

npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "build:css"
npm ERR! node v5.5.0
npm ERR! npm  v3.8.0
npm ERR! code ELIFECYCLE
npm ERR! pem@1.0.0 build:css: `npm run scss && npm run autoprefixer`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the pem@1.0.0 build:css script 'npm run scss && npm run autoprefixer'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the pem package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run scss && npm run autoprefixer
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs pem
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls pem
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Library/WebServer/Documents/pem/wp-content/themes/pem/npm-debug.log

npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "build:all"
npm ERR! node v5.5.0
npm ERR! npm  v3.8.0
npm ERR! code ELIFECYCLE
npm ERR! pem@1.0.0 build:all: `npm run build:css && npm run build:js && npm run build:images`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the pem@1.0.0 build:all script 'npm run build:css && npm run build:js && npm run build:images'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the pem package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run build:css && npm run build:js && npm run build:images
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs pem
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls pem
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Library/WebServer/Documents/pem/wp-content/themes/pem/npm-debug.log`
```

**Comment by Dexus** [comment]
Update your dependencies so that your scripts working with node v5.5.0 and watch your build scripts...

Pem has the current version 1.8.3

Closes #76

**Comment by Dexus** [comment]
Error by your own project https://github.com/zakirsajib/pem

please rename your package to resolve conflicts
